src/components/CodeScreen.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react'
+
+import { BackButton } from './Buttons'
+
+interface CodeScreenProps {
+  coinSelection: string
+  setShowCodeScreen: Function
+}
+
+const codeScreenStyle = {
+  textAlign: 'center' as const
+}
+
+const textStyle = {
+  fontSize: '40px',
+  marginTop: '36px'
+}
+
+export function CodeScreen(props: CodeScreenProps): JSX.Element {
+  const handleBackBtnClick = (): void => {
+    props.setShowCodeScreen(false)
+  }
+
+  return (
+    <>
+      <div style={codeScreenStyle}>
+        <div style={textStyle}>YOU'VE SELECTED {props.coinSelection}</div>
```

Comment:
Avoid 'literal strings'.
Create a constant json file and import from there.

Reply:
Updated per comment. A `constant.json` file was created in PR https://github.com/EdgeApp/edge-snack-bar-app/pull/2.